sopel/coretasks.py

```diff
@@ -539,6 +539,37 @@ def recieve_cap_ack_sasl(bot):
     bot.write(('AUTHENTICATE', mech))
 
 
+def send_authenticate(bot, token):
+    """Send ``AUTHENTICATE`` command to server with the given ``token``.
+
+    :param bot: instance of IRC bot that must authenticate
+    :param str token: authentication token
+
+    In case the ``token`` is more than 400 bytes, we need to split it and send
+    as many ``AUTHENTICATE`` command as needed. If the last chunk is 400 bytes
```

Comment:
```suggestion
    as many ``AUTHENTICATE`` commands as needed. If the last chunk is 400 bytes
```

I'm nitpicking docstrings, as usual. :joy_cat: